src/Aligent/AnnouncementBundle/Layout/DataProvider/AnnouncementDataProvider.php

```diff
@@ -120,75 +105,75 @@ public function getDisplayStatus(): bool
     /**
      * Returns the date config value for the provided date config key
      * @param string $dateKey - Date config key
-     * @return string
+     * @return Carbon|null
      */
-    private function getAlertBlockDateConfig(string $dateKey): string
+    protected function getAlertBlockDateConfig(string $dateKey): ?Carbon
     {
         // get config parameters
-        $website = $this->websiteManager->getCurrentWebsite();
-        $dateField = $this->configManager->get(
-            Configuration::getConfigKeyByName($dateKey),
-            false,
-            false,
-            $website
-        );
-        return $dateField ? $dateField->format('d-m-Y') : '';
+        $dateField = $this->getConfiguration($dateKey);
+
+        if (!$dateField instanceof \DateTime) {
+            return null;
+        }
+
+        return $this->toSystemTimeZone(new Carbon($dateField));
+    }
+
+    protected function toSystemTimeZone(Carbon $carbon): Carbon
+    {
+        $timeZone = new DateTimeZone($this->localeSettings->getTimeZone());
```

Comment:
I don't think this `DateTimeZone` is even needed, I think `$carbon->tz()` accepts a string anyway (ie `return $carbon->tz($this->localeSettings->getTimeZone())`) but it's only a minor improvement